### ISSUE TITLE
Esther versification doesn't match to vulg versification

### ISSUE THREAD

**Issue by LAfricain** [BeTheLight/ENG-B-CPDV2009-pd-PSFM#13]
Hello,
The CPDV versification of Esther doesn't match with the vulgate versification. Any solution? Specially for the CrossWire module building.

**Comment by DavidHaslam** [comment]
Spelling: **Esther**, not Eshter !

btw. **CrossWire** should always be in CamelCase.

**Comment by cmahte** [comment]
For building a sword module, I don't have any solution.  The CPDV uses it's
own versification for Song of Songs, & Esther. (and really the "canonical"
introductions to Sirach and Lamentations should be in a versification too.)

I built a ParaTEXT  versification file to be able to import the notes and
paragraphs from the Challoner revision. However, I don't see building
another versification file for the CPDV alone will apply in the sword
project, since it's not a file, but integral to the engine.

**Comment by cmahte** [comment]
If you do find differences from these files to the 'errata' page, please
report them here on github. I'm not currently online for fixing the files,
but I'm very interested in keeping them up to date.

**Comment by LAfricain** [comment]
Ok I will find a solution for Esther, but for the extra verse in SNG where it found it?? Because I can't see it in the Vulgate.
For the errata page you said that I need to apply only the correction propositions of 2018? Right?

**Comment by cmahte** [comment]
I just discovered the mte9 tags still have values in them.  The end titles
are from the html files for the Latin/English Diglot.

However, my non-standard tag mte9 is supposed to be empty, only to balance
columns. I plan to remove the visible end title for the print version. If
you choose to keep the end titles, these should be mte, as you have
suggested. I have not sought Ron's thoughts about whether the end titles
are just a locator (like a page header), or if they are important to his
translation. Howeer, he's seen and provided feedback on about 8 Books of
the Bible in sample PDF form, none of them had the end title visible.

I'll also note that some books (at least Sirach) have different text in the
leading title vs. the end title.  I hadn't deleted the text yet because I
haven't finalized which form should be the form used for print.

**Comment by LAfricain** [comment]
For mte9 I will change to mte. And about SNG extra verse?

**Comment by cmahte** [comment]
Corrections: - I've been through the list completely, including the 2018
stuff, but it's a good idea to check it, especially the 2018

I found words that didn't match between the English and the Latin-English
editions.  There were more than appear in the errata list for 2018, the
additional items were already listed in his errata list, but not applied to
all places they suggest, or to only one of the 2 editions.

Usually it was obvious which was correct. I fixed the error and reported it
to Ron.  Ron let me know when he'd looked over the issues, but not what
he'd done. It is possible that the "obvious" choice of one or the other
file may not be what Ron did.

**Comment by cmahte** [comment]
For Song of Solomon, I'm not sure what you mean 'extra verses'.

http://sacredbible.org/studybible/OT-24_Song.htm

This was the source text I used for Song of Songs. This file contains the
Latin text it is translated from, with Ron's versification applied to both
the Latin and the English. The verse numbers move to the points he thinks
different people are speaking.

As far as I know my text matches this file. I haven't lined the Latin in
that file up to the VULG module, but I expect the words are in order and
match, but the verse numbers move compared to the VULG module.

If I've failed to follow the verse numbers Ron applied, then we need to fix
the CPDV Text. If you think Ron is adding words.. I can't help with that.
If I didn't remove all the Latin and it is showing up as extra verses, I
need to fix that. I've looked through the file and it looks correct.
